packages/generator/templates/app/blitz.config.ts

```diff
@@ -15,5 +15,69 @@ const config: BlitzConfig = {
     return config
   },
   */
+  codegen:{
```

Comment:
Yeah after further thought, let's make this config default so that it doesn't have to be in blitz.config. This will be better for people upgrading too.